src/components/node-list/node-list.test.js

```diff
@@ -85,7 +86,13 @@ describe('NodeList', () => {
   });
 
   describe('visibility checkboxes on element items', () => {
-    const wrapper = setup.mount(<NodeList />);
+    const wrapper = setup.mount(<NodeList />, {
+      beforeLayoutActions: [() => toggleTypeDisabled('parameters', false)],
```

Comment:
similar to my above comment, good to mention that the parameters are only turned off for testing purposes here